packages/docs/src/components/base/Markdown.vue

```diff
@@ -116,7 +116,6 @@
       if (wantsList) code = code.map(c => `- ${c}\n`).join('')
 
       if (typeof code !== 'string') {
-        console.log(code, typeof code)
```

Comment:
I think this is reporting errors: d74a68cf

Reply:
Just reporting when code is anything other than an string. we can put it back if you want, only really removed it because it was outputting on render when building.

Reply:
Already put it back, it's reporting actual problems in the language files.